.github/workflows/workflow.yaml

```diff
@@ -16,6 +16,7 @@ jobs:
       matrix:
         platform:
           - { name: 'Linux',    arch: 'x64',        os: ubuntu-latest, werror: true }
+          - { name: 'Linux',    arch: 'arm64',      os: ubuntu-latest, werror: true }
```

Comment:
You need to install [gcc-aarch64-linux-gnu](https://packages.ubuntu.com/jammy/gcc-aarch64-linux-gnu) (and [g++-aarch64-linux-gnu](https://packages.ubuntu.com/jammy/g++-aarch64-linux-gnu)), and also use a CMake toolchain file.

The CMake toolchain file can be as simple as:
```cmake
set(CMAKE_SYSTEM_NAME "Linux")
set(CMAKE_SYSTEM_PROCESSOR "arm64")
set(CMAKE_C_COMPILER "aarch64-linux-gnu-gcc")
set(CMAKE_CXX_COMPILER "aarch64-linux-gnu-g++")
```

Reply:
I (ab)used my rights, and fixed it in d62cd4fd09d64cb82c58c2405888b25f4442e838

Reply:
ah thanks. Oops, I expected that an arm64 ubuntu box would run the action, but I realize now that arm runners are only in private beta. Thanks for adding the compilation!